tutorials/3d/environment_and_post_processing.rst

```diff
@@ -173,56 +321,28 @@ The tone mapping options are:
 
-Auto Exposure (HDR)
-^^^^^^^^^^^^^^^^^^^
-
-*This feature is only available when using the GLES3 backend.*
-
-Even though, in most cases, lighting and texturing are heavily artist controlled,
-Godot supports a basic high dynamic range implementation with the auto exposure
-mechanism. This is generally used for the sake of realism when combining
-interior areas with low light and outdoors. Auto exposure simulates the camera
-(or eye) in an effort to adapt between light and dark locations and their
-different amounts of light.
-
-.. image:: img/environment_hdr_autoexp.gif
-
-The simplest way to use auto exposure is to make sure outdoor lights (or other
-strong lights) have energy beyond 1.0. This is done by tweaking their **Energy**
-multiplier (on the Light itself). To make it consistent, the **Sky** usually
-needs to use the energy multiplier too, to match with the directional light.
-Normally, values between 3.0 and 6.0 are enough to simulate indoor-outdoor conditions.
-
-By combining Auto Exposure with *Glow* post processing (more on that below),
-pixels that go over the tonemap **White** will bleed to the glow buffer,
-creating the typical bloom effect in photography.
-
-.. image:: img/environment_hdr_bloom.png
-
-The user-controllable values in the Auto Exposure section come with sensible
-defaults, but you can still tweak them:
-
-.. image:: img/environment_hdr.png
-
-- **Scale:** Value to scale the lighting. Brighter values produce brighter images, smaller ones produce darker ones.
-- **Min Luma:** Minimum luminance that auto exposure will aim to adjust for. Luminance is the average of the light in all the pixels of the screen.
-- **Max Luma:** Maximum luminance that auto exposure will aim to adjust for.
-- **Speed:** Speed at which luminance corrects itself. The higher the value, the faster correction happens.
-
 Mid- and post-processing effects
 --------------------------------
 
 A large amount of widely-used mid- and post-processing effects are supported
-in the Environment.
+in the Environment resource.
+
+.. note::
+
+    Screen-space effects such as SSR, SSAO, SSIL and glow do not operate on
+    geometry that is located outside the camera view or is occluded by other
+    opaque geometry. Consider this when tweaking their settings to avoid
+    distracting changes during gameplay.
 
 Screen-Space Reflections (SSR)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-*This feature is only available when using the GLES3 backend.*
+*This feature is only available when using the Clustered Forward backend, not
+Forward Mobile or Compatibility.*
 
 While Godot supports three sources of reflection data (Sky, ReflectionProbe, and
-GIProbe), they may not provide enough detail for all situations. Scenarios
-where Screen Space Reflections make the most sense are when objects are in
+VoxelGI), they may not provide enough detail for all situations. Scenarios
+where screen-space reflections make the most sense are when objects are in
```

Comment:
Godot supports more than 3 sources of reflection data (Sky, ReflectionProbe, VoxelGI, SDFGI, LightmapGI (when using SH)), . I would generalize this to say Godot supports many sources of reflection data, but that SSR provides much higher detail when it works